#138 Update FAQ page 1.4.0 (#141)
Inbibo/adonisfx_docs: docs/faq.md

In order to add activation to the muscles it is necessary to define the fibers direction. This can be achieved by: first, painting the tendon weights to generate an initial estimation of fibers direction; and second, combing the fibers through the AdonisFX Paint Tool to customise the final fibers flow along the surface. Once the muscle has a valid fiber direction at each vertex, then the activation attribute triggers the contraction of the fibers. Ultimately, the level of activation can be modulated by an AdonisFX sensor to connect the contraction of the muscle to the animation of the rig. For more information about connecting sensors to muscles, please check this [section](maya/simple_setup#connect-sensors).

## How can I combine multiple activation values?

> Multiple activation values from different AdonisFX sensors (AdnSensorPosition, AdnSensorDistance, AdnSensorRotation) can be combined into a single value using the **AdnActivation** node. This node allows to override, add, subtract, multiply or divide multiple input activations to compute a single output value. For more details about the **AdnActivation** node, please refer to the [AdnActivation](maya/activation.md) page.

Reviewer: ```suggestion
Multiple activation values from different AdonisFX sensors (AdnSensorPosition, AdnSensorDistance, AdnSensorRotation) can be combined into a single value using the **AdnActivation** node. This node allows overriding, adding, subtracting, multiplying and dividing input activations to compute a single output value. For more details about the **AdnActivation** node, please refer to the [AdnActivation](maya/activation.md) page.
```

Reviewer 2: If you finally apply this change please also add it to the AdnActivation page

Author: Done.

## How can I add volume gain to the muscles?

The *Volume Ratio* attribute of an AdnMuscle allows to simulate volume gain (volume ratio greater than 1) and loss (volume ratio lower than 1). The influence of this attribute depends on the value of *Volume Preservation*, where 0 means no effect, and 1 means maximum effect.

## How can I simulate interaction between muscles?

AdonisFX provides two ways of simulating muscle-to-muscle interactions:

> - Using the AdnMuscle deformer. The AdnMuscle deformer allows to configure a muscle as a target for another muscle to simulate external geometry constraints (e.g. attachment to geometry and slide on geometry constraints). A muscle can be added as a target for another muscle in the same way as any other target geometry would be added. Detailed instructions for adding and removing targets can be found in this [section](maya/muscle.md#advanced).

Reviewer: ```suggestion
- Using the AdnMuscle deformer. The AdnMuscle deformer allows configuring geometry targets to simulate external constraints (e.g. attachment to geometry and slide on geometry constraints). A simulated muscle can be assigned as a target to another simulated muscle in the same way as any other geometry target would be added (e.g. bones). Detailed instructions for adding and removing targets can be found in this [section](maya/muscle.md#advanced).
```

Author: Done.

## How can I transfer the results of skin simulation to the final mesh?
Make use of the [AdnSkinMerge](maya/skin_merge) deformer to blend animated and simulated meshes into a single final mesh. Launch the Skin Merge tool, select the final mesh, add the animated and simulated target meshes and click on Create. Then you only need to paint the blend weight to define the influence of the simulated mesh targets over the animated ones. You can find more information [here](maya/simple_setup#adnskinmerge).

## How can I change the logger level?

> By default, AdonisFX solvers print log messages to the console if their level is `Info`, `Warning`, or `Error`. However, `Debug` log messages can be enabled to assist with troubleshooting. To enable `Debug` messages, set the environment variable `ADN_LOGGER_LEVEL` to `0`. To disable `Debug` messages again, set the environment variable to `1`. Note that for these changes to take effect, the AdonisFX plugin must be reloaded.

Reviewer: ```suggestion
AdonisFX Logger has different levels of execution: (0) Debug, (1) Info, (2) Warning, (3) Error. The level is determined by an environment variable `ADN_LOGGER_LEVEL`. If not defined, AdonisFX Logger Level is set to Info by default. This allows the system to print Info, Warning and Error messages to the console. If more verbosity is required, it is possible to modify the level to Debug. For that, the environment variable `ADN_LOGGER_LEVEL` has to be set to `0`. Note that for this change to take effect, the AdonisFX plugin must be reloaded.
```

Author: Done.

## How can I control the multithreading of AdonisFX solvers?

> AdonisFX nodes use multithreading based on a threshold that determines the minimum number of elements required for parallel processing. If the number of elements to process exceeds this threshold, multithreading will be applied, otherwise, the code will execute sequentially. This threshold is set to `16` by default, but can be adjusted by modifying the value of the `ADN_MIN_PARALLEL_SIZE` environment variable. Modify this value carefully, as it can improve performance in some cases but may also reduce it depending on your CPU. Note that for these changes to take effect, the AdonisFX plugin must be reloaded.

Reviewer: I think this is not true... This variable is read at each initialization. Could you please double
`Note that for these changes to take effect, the AdonisFX plugin must be reloaded.`

Reviewer 2: Carlos is correct. ADN_LOGGER_LEVEL gets read on load. But in this case it's on init

Author: Correct, I've double checked it and, ADN_MIN_PARALLEL_SIZE can be modified in the script editor (with python) and it will be read on init.
